pkg/checkpoint/checkpoint.go

```diff
@@ -173,37 +174,38 @@ func (sc *StableCheckpoint) Verify(
 	params *issconfig.ModuleParams,
 	hashImpl crypto.HashImpl,
 	chkpVerifier Verifier,
+	membership *trantorpbtypes.Membership,
 	logger logging.Logger,
```

Comment:
In fact, I would completely remove the logger from here (see later comments - added this one after them).

Reply:
Fair, done!